src/python/bot/fuzzers/libfuzzer_engine.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2019 Google LLC
```

Comment:
Looks like a copy of `src/python/bot/fuzzers/libFuzzer/engine.py` ?

Reply:
Removed!